src/Storages/MergeTree/MergeTreeIndexFullText.cpp

```diff
@@ -477,11 +477,11 @@ bool MergeTreeConditionFullText::traverseASTEquals(
         if (function.getFunctionName() == "arrayElement")
         {
             /** Try to parse arrayElement for mapKeys index.
-              * It is important to ignore keys like column_map['Key'] = '' because if key does not exists in map
+              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in map
               * we return default value for arrayElement.
```

Comment:
```suggestion
              * It is important to ignore keys like column_map['Key'] = '' because if key does not exist in the map
              * we return default the value for arrayElement.
```